packages/@okta/i18n/package.json

```diff
@@ -1,28 +1,28 @@
 {
-  "_from": "@okta/i18n@47.4.0",
-  "_id": "@okta/i18n@47.4.0",
+  "_from": "@okta/i18n@48.0.0",
```

Comment:
should the version number be "47.5.0"? This is the number I got from the release job log.

Reply:
@ganeshsomasundaram-okta did you promote i18n to get a release version? We should use the release version of i18n in the widget.

Reply:
yes, Mau helped me promote it. isn't 48.0.0 a release version? I saw the "beta" in the non-release package.

Reply:
Yes, `48.0.0` should be good.